faq.html

```diff
@@ -99,31 +99,34 @@ <h3 id='firewalls'>14. I'm behind a firewall &mdash; can I still log into Jabber
 <p>Yes, you can, but you need to make sure that outbound port 5222 is open (this is the IANA-registered port for XMPP client connections). To check whether your connection is blocked, type 'telnet jabber.org 5222' in a terminal or console window.</p>
 
 <h3 id='network'>15. Can I chat with people on other messaging services?</h3>
-<p>Yes! The Jabber.org service connects with all messaging services that use <a href='https://xmpp.org/'>XMPP</a>, the open standard for instant messaging and presence over the Internet. However, you can't connect from Jabber.org to proprietary services like Facebook, Skype, WhatsApp, or Discord because they use their own proprietary protocols.</p>
+<p>Yes! The Jabber.org service connects with all messaging services that use <a href='https://xmpp.org/'>XMPP</a>, the open standard for instant messaging and presence over the Internet. However, you can't connect from Jabber.org to proprietary services like Facebook, Skype, WhatsApp, or Discord (without a <a href='#gateways'>gateway</a> ) because they use their own proprietary protocols.</p>
 
-<h3 id='chatrooms'>16. Does Jabber.org support chatrooms?</h3>
+<h3 id='gateways'>16. Are there any gateways, transports, or bridges?</h3>
+<p>None hosted as part of the Jabber.org service, but the broader nework has many options for these to connect to various legacy and proprietary networks.</p>
```

Comment:
Thanks for the PR! Can we perhaps add instructions on how to find gateways on the "broader network"? (I haven't used them in years.)